recipes/mysql-server.rb

```diff
@@ -35,11 +35,13 @@
         action [:create, :start]
     end
 
-    mysql_config 'default' do
-        source 'mysql_config.erb'
-        notifies :restart, 'mysql_service[default]'
-        action :create
-    end
+    if node['mysql']['server_version'] != "5.5"
```

Comment:
@aligenttimaslat should there be a different erb file for a 5.5 install rather than none at all?